hack/update_owners.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+import re
+import json
+import os
+import sys
+import time
+import urllib2
+import zlib
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+OWNERS_PATH = os.path.abspath(os.path.join(BASE_DIR, '..', 'test', 'test_owners.csv'))
```

Comment:
nit: can we keep lines under 80 chars? Also will you put a line break so `BASE_DIR...test_owners.csv` all fits on the same line?